docs/scos/dev/upgradability-services/new-documentation/evaluator-error-description-table.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Evaluator errors reference page
+description: Evaluator error list with the descriptions
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Evaluator errors reference page
+
+This error page provides information about the error that is may occur while evaluator validation.
+Current page can be used to search for needed documentation faster and in most appropriate way. 
+
+#### Reference table usage flow
+
+To find needed documentation for your problem need to find check name that you have a problem
+
+Name of the specific check can be found at the Evaluation output logs. Check name placed at the beginning of each error log. 
+For example
+```bash
+-------------------- ----------------------------------------------------------------------------------------------------
+PrivateApi:Extension Please avoid extension of the PrivateApi Spryker\Zed\CustomerAccessGui\Communication\Form\CustomerAccessForm in Pyz\Zed\CustomerAccessGui\Communication\Form\CustomerAccessForm
+-------------------- ----------------------------------------------------------------------------------------------------
+```
+At this particular log example check name equal "PrivateApi:Extension"
+
+#### Reference table for evaluation check names and documentation pages links 
+| Check name  | Error message template | Link to Documentation page |
+| ----------- | ----------- | ----------- |
+| NotUnique:TransferName | Transfer object name **{transfer_name}** has to have project prefix Pyz in **{absolute_transfer_path}**, like **Pyz{transfer_name}** |  {link - unique-entity-name-not-unique.md#Making transfer names unique}  |
+| NotUnique:TransferProperty | Transfer property **{transfer_property_name}** for **{transfer}** has to have project prefix Pyz in **{absolute_transfer_path}**, like **pyz{transfer_property_name}** | {link - unique-entity-name-not-unique.md#Making transfer property names unique} |
```

Comment:
Can we separate them by groups with small description per group?

Reply:
Yes, let's do this after release.
This page was more like a template, for error codes